hive/src/main/java/org/apache/iceberg/hive/HiveCatalog.java

```diff
@@ -89,10 +91,13 @@ public HiveCatalog(String name, String uri, int clientPoolSize, Configuration co
 
     try {
       List<String> tables = clients.run(client -> client.getAllTables(database));
-      return tables.stream()
+      List<TableIdentifier> tableIdentifiers = tables.stream()
           .map(t -> TableIdentifier.of(namespace, t))
           .collect(Collectors.toList());
 
+      LOG.debug("Listing of namespace: {} resulted in the following tables: {}", namespace, tableIdentifiers);
```

Comment:
Let's clean up the message a bit: "Listing namespace {} returned tables: {}"

Reply:
Done